### DISCUSSION TITLE
input for better app/client (ios user)

### DISCUSSION THREAD

**Discussion by dangdingdong1980** [advplyr/audiobookshelf-app#527] [Ideas]
![image5](https://user-images.githubusercontent.com/61085413/212539239-8d102cef-5cd7-4ec2-aadd-244b9e6157ae.jpeg)
![image4](https://user-images.githubusercontent.com/61085413/212539241-f8a71adf-a15a-4d37-ab60-9f55764a8099.jpeg)
![image3](https://user-images.githubusercontent.com/61085413/212539243-5336414a-63e5-4e75-a89c-3fe6932c952d.jpeg)
Hi, i have some input if it's possible to be considered (for ios client) 👍 

PS : I use Prologue as comparison. What I mention below all available on Prologue app

1. at the home tab - continue listening / recently added / series / author to list out more than current display ? (currently if i'm not wrong only limited to 5 or 10 item per list)

2. and for the library tab - it would be nice if we have alphabet index at the right side, so we can simply touch the alphabet and jump to the one we're looking for, rather than scroll alll the way down to find the books/author

3. on the library card - it would be great if it has the Narrator information as well. Because Narrator is important for me, maybe for others as well

4. currently we have tab : home, library, series and collection. possible to add Author and Narrator tab ? ( I know there's filter for that, but for shortcut will be great)

5. on the playing screen : Narrator information right below the Author will be great. Because for me personally, If I like the Narrator, I would want to listen to other books with the same Narrator

6. And the playing position, instead of at the bottom of the screen, maybe it will be nicer and more comfortable to use if we put it above the menu bar - after the book cover)

**Comment by gh0sti** [comment]
So I believe prologue app dev is going to look into connecting to audiobookshelf api which allow us to use prologue to connect to our shelf servers.

**Comment by KillzMckinzie** [comment]
@gh0sti You are correct. You can see Prologues roadmap here - https://trello.com/c/LYcalqZM/47-audiobookshelf-integration

**Comment by advplyr** [comment]
Number 6 on your list is implemented for the next release which will be 0.9.61-beta

**Comment by KaiStarkk** [comment]
I’d also love to have the author tab on the app. The information is already there on the server as the view is available on the web version. 

I have almost 2000 audiobooks in the server so I browse by author rather than book, and that experience is better on the web with pictures and descriptions rather than as a filter list

**Comment by gh0sti** [comment]
Would it be possible to put the author tab @advplyr ?

**Reply by advplyr** [reply]
Yes, it will be added
